Added support for long long and unsigned long long

ankane opened the pull request:
Added automatic conversion of `long long` and `unsigned long long` types.

ankane commented:
Updated `.travis.yml` since latest rake and minitest require Ruby 2.2+. May be worth considering dropping support for [EOL Ruby versions](https://www.ruby-lang.org/en/downloads/branches/) (2.3 and below).

jasonroelofs commented:
This is great, thanks! Feel free to drop support for older Rubies. I'd rather we follow Ruby's own EOL schedule than have hard-coded versions like this that will get old real quick.

ankane commented:
Dropped support for EOL Ruby's. Also moved to bundler to make contributing more intuitive since it's now the standard way to manage gems (can revert and discuss in a separate issue if you have concerns).
